python/ct/crypto/verify.py

```diff
@@ -11,6 +11,44 @@
 from ct.proto import ct_pb2
 import ecdsa
 
+def decode_signature(signature):
+    """Decode the TLS-encoded serialized signature.
+
+    Args:
+        signature: TLS-encoded signature.
+
+    Returns:
+        the inner, ASN.1 encoded contents.
```

Comment:
I think not - that's the original method description - but I'll be happy to change.
Also, it's wrong :) so I'll fix